### PR TITLE
docs(redis) update redis chapter

### PR DESCRIPTION
**Pull request by kamilmysliwiec**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

## Other information

### CODE REVIEW

File: content/microservices/redis.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
 ### Redis
 
-A second built-in transporter is based on [Redis](https://redis.io/) database. This transporter takes advantage of **publish/subscribe** feature.
+[Redis](https://redis.io/) transporter implements the publish/subscribe messaging paradigm and leverages [Pub/Sub](https://redis.io/topics/pubsub) feature of Redis. Published messages are categorised in channels, without knowing what subscribers will eventually receive the message (and if any). Each microservice can subscribe to any number of channels. In addition, more than one channel can be subscribed to at a time. Messages exchanged through channels are **fire-and-forget**, which means that if a message is published and there are no subscribers interested in it, the message removed and cannot be recovered. Thus, you don't have a guarantee that either messages or events will be handled by at least one service. Also, single message can be received many subscribers.
```
Comment (johnbiundo) [inline content/microservices/redis.md]: The [Redis](https://redis.io/) transporter implements the publish/subscribe messaging paradigm and leverages the [Pub/Sub](https://redis.io/topics/pubsub) feature of Redis. Published messages are categorized in channels, without knowing what subscribers (if any) will eventually receive the message. Each microservice can subscribe to any number of channels. In addition, more than one channel can be subscribed to at a time. Messages exchanged through channels are **fire-and-forget**, which means that if a message is published and there are no subscribers interested in it, the message is removed and cannot be recovered. Thus, you don't have a guarantee that either messages or events will be handled by at least one service. A single message can be subscribed to (and received) by multiple subscribers.

File: content/microservices/redis.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
+To start building Redis-based microservices, first install the required package:
 
 ```bash
 $ npm i --save redis
 ```
 
 #### Overview
 
-In order to switch from TCP transport strategy to Redis **pub/sub**, we need to change an options object passed to the `createMicroservice()` method.
+In order to use Redis transporter, a different options object has to be passed to the `createMicroservice()` method.
```
Comment (johnbiundo) [inline content/microservices/redis.md]: To use the Redis transporter, pass the following options object to the `createMicroservice()` method:

File: content/microservices/redis.md [hunk 3]
```diff
@@ -26,11 +26,47 @@ const app = await NestFactory.createMicroservice(ApplicationModule, {
 });
 ```
 
-> info **Hint** `Transport` enumerator is imported from the `@nestjs/microservices` package.
+> info **Hint** `Transport` enum is imported from the `@nestjs/microservices` package.
```
Comment (johnbiundo) [inline content/microservices/redis.md]: The `Transport` enum... <- add *The*

File: content/microservices/redis.md [hunk 4]
```diff
@@ -26,11 +26,47 @@ const app = await NestFactory.createMicroservice(ApplicationModule, {
...
+
+Likewise, to create a client instance, we need to pass the options object which has the same properties we saw above in the `createMicroservice()` method.
```
Comment (johnbiundo) [inline content/microservices/redis.md]: Likewise, to create a client instance, we need pass an options object with the same properties we saw above in the `createMicroservice()` method.

File: content/microservices/redis.md [hunk 5]
```diff
@@ -26,11 +26,47 @@ const app = await NestFactory.createMicroservice(ApplicationModule, {
...
+    }
+  },
+]),
+```
+
+Other options to create a client (either `ClientProxyFactory` or `@Client()`) can be used as well. You can read about them [here](/microservices/basics#client).
+
+#### Context
+
+In more sophisticated scenarios, you may want to access more information about the incoming request. In Redis, you can access `RedisContext` object.
```
Comment (johnbiundo) [inline content/microservices/redis.md]: ...you can access the `ReisContext`... <- add *the*

File: content/microservices/redis.md [hunk 6]
```diff
@@ -39,10 +75,10 @@ There are a bunch of available options that determine a transporter behaviour.
   </tr>
   <tr>
     <td><code>retryAttempts</code></td>
-    <td>A total amount of connection attempts</td>
+    <td>Number of times to retry messag</td>
```
Comment (johnbiundo) [inline content/microservices/redis.md]: message <- fix typo

File: content/microservices/redis.md (hunk 1 above)
Comment (kamilmysliwiec) [inline content/microservices/redis.md]:
```suggestion
[Redis](https://redis.io/) transporter implements the publish/subscribe messaging paradigm and leverages [Pub/Sub](https://redis.io/topics/pubsub) feature of Redis. Published messages are categorized in channels, without knowing what subscribers (if any) will eventually receive the message. Each microservice can subscribe to any number of channels. In addition, more than one channel can be subscribed to at a time. Messages exchanged through channels are **fire-and-forget**, which means that if a message is published and there are no subscribers interested in it, the message is removed and cannot be recovered. Thus, you don't have a guarantee that either messages or events will be handled by at least one service. A single message can be subscribed to (and received) by multiple subscribers.
```

File: content/microservices/redis.md [hunk 7]
```diff
@@ -1,20 +1,20 @@
...
+To start building Redis-based microservices, first install the required package:
 
 ```bash
 $ npm i --save redis
 ```
 
 #### Overview
 
-In order to switch from TCP transport strategy to Redis **pub/sub**, we need to change an options object passed to the `createMicroservice()` method.
+In order to use Redis transporter, a different options object has to be passed to the `createMicroservice()` method.
```
Comment (kamilmysliwiec) [inline content/microservices/redis.md]:
```suggestion
To use the Redis transporter, pass the following options object to the `createMicroservice()` method:
```

File: content/microservices/redis.md (hunk 3 above)
Comment (kamilmysliwiec) [inline content/microservices/redis.md]:
```suggestion
> info **Hint** The `Transport` enum is imported from the `@nestjs/microservices` package.
```

File: content/microservices/redis.md [hunk 8]
```diff
@@ -26,11 +26,47 @@ const app = await NestFactory.createMicroservice(ApplicationModule, {
 });
 ```
 
-> info **Hint** `Transport` enumerator is imported from the `@nestjs/microservices` package.
+> info **Hint** The `Transport` enum is imported from the `@nestjs/microservices` package.
+
+Likewise, to create a client instance, we need to pass the options object which has the same properties we saw above in the `createMicroservice()` method.
```
Comment (kamilmysliwiec) [inline content/microservices/redis.md]:
```suggestion
Likewise, to create a client instance, we need to pass an options object with the same properties we saw above in the `createMicroservice()` method.
```

File: content/microservices/redis.md [hunk 9]
```diff
@@ -26,11 +26,47 @@ const app = await NestFactory.createMicroservice(ApplicationModule, {
...
+    }
+  },
+]),
+```
+
+Other options to create a client (either `ClientProxyFactory` or `@Client()`) can be used as well. You can read about them [here](/microservices/basics#client).
+
+#### Context
+
+In more sophisticated scenarios, you may want to access more information about the incoming request. In Redis, you can access `RedisContext` object.
```
Comment (kamilmysliwiec) [inline content/microservices/redis.md]:
```suggestion
In more sophisticated scenarios, you may want to access more information about the incoming request. In Redis, you can access the `RedisContext` object.
```

File: content/microservices/redis.md (hunk 6 above)
Comment (kamilmysliwiec) [inline content/microservices/redis.md]:
```suggestion
    <td>Number of times to retry message</td>
```

**Pull request merged**